.gitignore

```diff
@@ -17,4 +17,5 @@ dist
 natlas-server/media
 natlas-server/logs
 natlas-agent/logs
-tmp
\ No newline at end of file
+tmp
+myscopefile.txt
```

Comment:
Is this general to all users? This seems like it's specific the name that you used when you created this file.

Reply:
Yeah the existence of tmp is so that any local development files can be placed there without needing to add it to the overall repo. Other files in the pull request are hard coding references to this file. We should not do that. We could write ways to access to the ./add-scope.py from outside the docker container, but we should not, under any circumstances, be adding scope to deployments or be referencing automatic addition of scope to deployments. Scope is very deployment specific and does not need to be included in deployment automation.